apps/webapp/app/components/navigation/Breadcrumb.tsx

```diff
@@ -1,16 +1,20 @@
-import { RouteMatch, useMatches } from "@remix-run/react";
+import { UIMatch, useMatches, Params } from "@remix-run/react";
 import { Fragment, ReactNode } from "react";
 import { BreadcrumbIcon } from "../primitives/BreadcrumbIcon";
+import { Handle } from "~/utils/handle";
 
-export type BreadcrumbItem = (match: RouteMatch, allMatches: RouteMatch[]) => ReactNode;
+export type BreadcrumbItem = (
+  match: UIMatch<unknown, Handle>,
+  allMatches: UIMatch<unknown, Handle>[]
+) => ReactNode;
 
 export function Breadcrumb() {
-  const matches = useMatches();
+  const matches: UIMatch<unknown, Handle>[] = useMatches() as UIMatch<unknown, Handle>[];
```

Comment:
Why is this being cast? We should avoid doing this kind of thing unless it's absolutely necessary